File: Tecnicell.Server/Models/Responses/ScreenResponse.cs
using Tecnicell.Server.Models.Entity;
using Tecnicell.Server.Models.ViewModel.Phone;
using Tecnicell.Server.Models.ViewModel;
using Tecnicell.Server.Models.ViewModel.Battery;
using Tecnicell.Server.Models.ViewModel.Accessory;
using Tecnicell.Server.Models.ViewModel.Screen;

namespace Tecnicell.Server.Models.Responses
{
    public class ScreenResponse
    {
        public ScreenView? View { get; set; }
        public ImageViewModel? Image { get; set; }
        public IEnumerable<ScreenHistoryViewModel> Histories { get; set; } = new List<ScreenHistoryViewModel>();
    }
}

File: tecnicell.client/src/app/Interfaces/business/ApiResponses/ScreenResponse.ts
import { ScreenHistory } from "../Models/Screen"
import { Image } from "../Models/Image";

export interface ScreenResponse {
    view:      ScreenView;
    image:     Image;
    histories: ScreenHistory[];
}
export interface ScreenView {
    code:      string;
    type:      string;
    name:      string;
    salePrice: number;
    imageCode: string;
    quantity:  number;
    warranty:  number;
    available: string;
}
